### ISSUE TITLE
The ddl files for PostgreSQL is missing as the DB to use for the EJB timer service.

### ISSUE THREAD

**Issue by endtak** [eclipse-ee4j/glassfish#22850]
## Environment Details
* GlassFish Version (and build number):5.1
* JDK version:1.8
* OS:Windows 10

----------

## Problem Description
The ddl files for PostgreSQL is missing as the DB to use for the EJB timer service.
```
C:\glassfish5>dir /b C:\glassfish5\glassfish\lib\install\databases
dbvendormapping
ejbtimer_db2.sql
ejbtimer_derby.sql
ejbtimer_mssqlserver.sql
ejbtimer_mysql.sql
ejbtimer_oracle.sql
ejbtimer_pointbase.sql
ejbtimer_sybase.sql
ejbtimer_symfoware.sql
jsr352-db2.sql
jsr352-derby.sql
jsr352-mssqlserver.sql
jsr352-mysql.sql
jsr352-oracle.sql
jsr352-postgresql.sql
jsr352-sybase-ace.sql
oracle
upgrade
```

**Comment by SplotyCode** [comment]
This can be closed as this was solved in #22858 right?

**Comment by endtak** [comment]
Thx! Close this issue.
